client/src/cards/AlertLevel/index.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import {q} from "@client/context/AppContext";
+import Button from "@thorium/ui/Button";
+
+const AlertLevel = () => {
+    const [ship] = q.ship.get.useNetRequest();
+    let [description, setDescription] = React.useState("");
+    const alertLevelText = [
+      {
+        number: 5,
+        text:
+          "This alert condition is used when the ship is at normal running status. The crew is on standard duty and the ship is in no danger.",
+      },
+      {
+        number: 4,
+        text:
+          "This alert condition is used when the station has a minor problem. All crew except damage control is on standard duty.",
+      },
+      {
+        number: 3,
+        text:
+          "This alert condition is used when the ship needs to be ready for a crisis. All off duty personnel are put on stand by status.",
+      },
+      {
+        number: 2,
+        text:
+          "This alert condition is used when the ship is in a dangerous situation, but is safe for the moment. All crew members are put on duty.",
+      },
+      {
+        number: 1,
+        text:
+          "This alert condition is used when the ship is in danger or under attack. All crew members are put on duty at battle stations.",
+      },
+    ];
+
+    const updateLevel = (newLevel: string) => {
+      q.alertLevel.update.netSend({
+        level: newLevel
+      });
+    }
+    const displayDesc = (level: number) => {
+      alertLevelText.forEach((e) => {
```

Comment:
Another way to do this would be changing the `alertLevelText` array into an object where they keys are the alert level and the value is the text. That way you could just access the text for each level directly.